src/components/organisms/StorageTable/index.jsx

```diff
@@ -99,6 +100,14 @@ const StorageTable = () => {
                           <GridItem span={6}>
                             <GraphRow doc={doc} />
                           </GridItem>
+                          <GridItem span={12}>
+                            <div style={{ marginTop: "16px", display: "flex", justifyContent: "flex-end" }}>
```

Comment:
PatternFly generally discourages hard-coded spacing like marginTop: "16px".
Suggest using PF spacing utilities (pf-v5-u-mt-md) or global spacer tokens for consistency and maintainability.